ptd_client_server/lib/source_hashes.py

```diff
@@ -57,11 +57,7 @@ def init() -> None:
     result["sources"] = get_sources_checksum(base_dir)
 
     for name, module in sys.modules.items():
-        if not (
-            name == "__main__"
-            or name == _module_name
-            or name.startswith(_module_name + ".")
-        ):
+        if not (name == _module_name or name.startswith(_module_name + ".")):
```

Comment:
I had to remove "__main__" from this second check because if the package is installed and client/server run somewhere from .../bin/ it won't be under base_dir.